src/PrestaShopBundle/Install/Install.php

```diff
@@ -898,9 +898,9 @@ public function getModulesList()
             'blockwishlist',
             'contactform',
             'dashactivity',
+            'dashtrends',
```

Comment:
Waoh 😮  so this list indirectly controls what modules is hooked first or last, which impacts the display of the BO ?!